### PR TITLE
Get recipes route

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Implments list recipes route as described by the OpenApi spec

### CODE REVIEW

**Review by jonmattgray — APPROVED** [review]

File: test/integration/recipe.test.js [hunk 1]
```diff
@@ -33,22 +34,7 @@ describe('Recipes', function () {
...
-        .get('/v1/members/invalid')
-        .reply(404, {})
-        .get('/v1/members/error')
-        .reply(500, {})
-    })
-
-    afterEach(async function () {
-      nock.cleanAll()
-    })
+    setupIdentityMock()
```
Comment (n3op2) [inline test/integration/recipe.test.js:37]: it kinda defeats the integration e2e being e2e. Anyway, we have made a head start with this approach. I think at some point once we have covered all the endpoints. We can start thinking of spinning up all services (github actions/flows) and running these tests without needing to mock or anything. It also could be something like a postman collection that could be exported.

I think we should start the service before running these so express app does not need to be initiated this way.

File: test/integration/recipe.test.js [hunk 2]
```diff
@@ -59,7 +45,7 @@ describe('Recipes', function () {
         alloy: 'foobar3000',
         price: 'foobar3000',
         requiredCerts: [{ description: 'foobar3000' }],
-        supplier: 'foobar3000',
+        supplier: 'valid-1',
```
Comment (n3op2) [inline test/integration/recipe.test.js:48]: looks like `newRecipe` repeats itself a fdew times, maybe this could be moved in terms of scope? (fixture)

**Review by n3op2 — APPROVED** [review]

File: test/integration/recipe.test.js (hunk 2 above)
Comment (mattdean-digicatapult) [reply test/integration/recipe.test.js:48]: Yeah this was part of Dan's implementation. Agree it's not great

File: test/integration/recipe.test.js (hunk 1 above)
Comment (mattdean-digicatapult) [reply test/integration/recipe.test.js:37]: Depends on what we mean by integration. At this point you could call these service tests maybe? We will indeed very similar e2e integration tests, but as you said the goal here is to keep us moving whilst we have auth issues

**Pull request merged**